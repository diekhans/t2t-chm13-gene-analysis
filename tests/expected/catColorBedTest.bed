chr1	14421	21268	CHM13_T0000001	0	+	21268	21268	0,0,0	2	73,534,	0,6313,
chr1	20691	137457	CHM13_T0000003	0	-	137457	137457	0,0,0	4	565,180,33,98,	0,7924,104626,116668,
chr1	112071	113065	CHM13_T0000004	0	-	112108	113046	0,0,0	2	596,398,	0,596,
chr1	148776	151582	CHM13_T0000005	0	+	151582	151582	0,0,0	2	2263,186,	0,2620,
chr1	156419	191953	CHM13_T0000007	0	+	191953	191953	0,0,0	16	64,51,197,52,197,56,193,234,64,67,190,103,118,65,84,523,	0,157,2269,3565,5675,6971,9090,10061,10388,13549,14460,14986,16982,26359,29324,35011,
chr1	184932	205394	CHM13_T0000013	0	-	205394	205394	0,0,0	8	893,525,59,66,216,132,110,405,	0,6496,8102,10321,11270,14749,15944,20057,
chr1	187616	188695	CHM13_T0000015	0	+	188695	188695	0,0,0	2	244,212,	0,867,
chr1	246546	249013	CHM13_T0000016	0	+	249013	249013	0,0,0	2	832,1115,	0,1352,
chr1	255346	256663	CHM13_T0000018	0	-	256663	256663	0,0,0	1	1317,	0,
chr1	256738	283560	CHM13_T0000026	0	-	283560	283560	0,0,0	6	176,102,180,100,134,851,	0,1402,24324,25064,25788,25971,
chr1	256744	288224	CHM13_T0000030	0	+	288224	288224	0,0,0	8	172,102,3301,153,184,96,1745,2637,	0,1398,13591,20047,24320,25064,25784,28843,
chr1	289368	295863	CHM13_T0000056	0	+	295863	295863	0,0,0	2	1048,1461,	0,5034,
chr1	297213	306990	CHM13_T0000058	0	-	306990	306990	0,0,0	3	605,1044,158,	0,6042,9619,
chr1	302435	303493	CHM13_T0000061	0	+	303493	303493	0,0,0	1	1058,	0,
chr1	303672	307321	CHM13_T0000062	0	-	307321	307321	0,0,0	5	63,109,100,85,480,	0,518,997,2941,3169,
chr1	334991	345789	CHM13_T0000063	0	+	345789	345789	0,0,0	2	124,227,	0,10571,
chr1	341237	344762	CHM13_T0000064	0	+	344762	344762	0,0,0	2	39,3015,	0,510,
chr1	343972	344784	CHM13_T0000065	0	+	344784	344784	0,0,0	2	274,168,	0,644,
chr1	346682	349505	CHM13_T0000067	0	-	349505	349505	0,0,0	4	851,89,91,358,	0,1152,1955,2465,
chr1	355532	373485	CHM13_T0000071	0	+	355743	373064	0,0,0	14	70,92,182,51,125,90,186,163,116,79,500,125,111,667,	0,191,4425,5309,9200,12429,12664,14525,15517,15791,15940,16634,17076,17286,
chr1	372461	389762	CHM13_T0000086	0	-	389762	389762	0,0,0	20	255,896,90,136,114,144,102,114,112,140,189,114,111,79,91,121,132,175,153,1594,	0,354,1506,1967,2622,2851,4580,4939,7619,8492,8904,9667,10274,10496,12415,12587,13386,13591,15421,15707,
chr1	389542	394599	CHM13_T0000094	0	+	389652	394075	0,0,0	12	217,260,122,222,117,214,145,168,89,74,182,753,	0,709,1045,1242,1771,2120,2525,2753,3336,3523,3765,4304,
chr1	392294	394598	CHM13_T0000099	0	-	394598	394598	0,0,0	4	672,69,185,755,	0,770,1007,1549,
chr1	395361	404743	CHM13_T0000101	0	+	395411	403453	0,0,0	16	133,100,147,81,73,128,96,81,76,137,150,141,141,219,49,1424,	0,222,3794,4038,4203,4396,4630,4841,5591,5804,6377,6702,7016,7349,7832,7958,
chr1	404082	411038	CHM13_T0000107	0	-	405049	411038	0,0,0	3	1065,126,2149,	0,1294,4807,
chr1	426014	428318	CHM13_T0000109	0	-	428318	428318	0,0,0	2	383,1723,	0,581,
chr1	429285	430496	CHM13_T0000110	0	-	429382	430297	0,0,0	3	471,88,481,	0,564,730,
chr1	431461	444840	CHM13_T0000114	0	+	444304	444778	0,0,0	3	144,86,557,	0,7033,12822,
chr1	438376	438530	CHM13_T0000116	0	-	438530	438530	0,0,0	1	154,	0,
chr1	442292	443492	CHM13_T0000117	0	-	443492	443492	0,0,0	2	194,280,	0,920,
chr1	464410	486350	CHM13_T0000121	0	+	464860	485215	0,0,0	38	598,48,216,225,213,207,219,195,201,149,106,117,165,144,125,106,339,138,128,115,120,354,193,216,230,135,97,165,112,12,117,193,88,24,33,225,104,1293,	0,1170,6560,7057,7362,7793,9074,9373,9658,9944,10169,10996,11194,11568,11799,11995,12232,12655,13162,13408,13611,13847,14689,15057,15387,15724,16054,16248,16547,16853,17074,17274,17549,17814,19284,19576,20271,20647,
chr1	479121	486350	CHM13_T0000128	0	-	486350	486350	0,0,0	13	166,221,229,134,98,163,114,115,193,88,225,107,1294,	0,341,676,1012,1341,1536,1833,2362,2560,2836,4863,5556,5935,
chr1	485266	486350	CHM13_T0000130	0	+	486350	486350	0,0,0	2	183,219,	0,865,
chr1	492422	493503	CHM13_T0000134	0	-	493503	493503	0,0,0	2	601,228,	0,853,
chr1	493293	499618	CHM13_T0000135	0	+	499618	499618	0,0,0	10	123,87,192,100,124,81,81,135,148,148,	0,401,1129,1409,2242,3477,4456,4759,5467,6177,
chr1	501229	504569	CHM13_T0000139	0	-	502079	502829	0,0,0	2	1609,291,	0,3049,
chr1	510598	515989	CHM13_T0000140	0	+	515989	515989	0,0,0	5	1171,30,29,136,65,	0,2536,2663,4064,5326,
chr1	510599	544865	CHM13_T0000141	0	-	511674	520326	0,0,0	12	1170,469,26,135,66,96,76,108,94,113,54,22,	0,2097,2663,4061,5322,5685,8536,9059,9655,10174,33206,34244,
chr1	526367	533381	CHM13_T0000160	0	+	533381	533381	0,0,0	2	529,566,	0,6448,
chr1	565801	572840	CHM13_T0000161	0	+	572840	572840	0,0,0	3	94,208,1318,	0,3807,5721,
chr1	595869	595964	CHM13_T0000162	0	+	595964	595964	0,0,0	1	95,	0,
chr1	596628	596718	CHM13_T0000163	0	+	596718	596718	0,0,0	1	90,	0,
chr1	597770	597853	CHM13_T0000164	0	+	597853	597853	0,0,0	1	83,	0,
chr1	598122	599109	CHM13_T0000165	0	+	599109	599109	0,0,0	1	987,	0,
chr1	601821	608321	CHM13_T0000166	0	-	608321	608321	0,0,0	2	3341,191,	0,6309,
chr1	602645	626486	CHM13_T0000168	0	+	607981	626397	0,0,0	16	47,56,66,145,81,307,119,130,75,86,172,172,141,117,94,499,	0,386,544,5309,5777,6154,6603,6851,7861,8481,8996,10040,10873,22504,22997,23342,
chr1	632133	635218	CHM13_T0000176	0	-	632469	635197	0,0,0	5	461,203,88,123,208,	0,526,891,1862,2877,
chr1	639965	642779	CHM13_T0000179	0	-	640180	642753	0,0,0	7	286,129,197,67,102,123,171,	0,364,602,1298,1652,2323,2643,
chr1	645531	660693	CHM13_T0000183	0	+	660693	660693	0,0,0	7	777,183,159,111,139,480,176,	0,1545,1863,6338,6923,11561,14986,
chr1	645534	660693	CHM13_T0000186	0	-	646295	657419	0,0,0	7	896,176,159,114,137,479,176,	0,1549,1860,6335,6923,11559,14983,
chr1	660862	663667	CHM13_T0000193	0	+	660904	661894	0,0,0	1	2805,	0,
chr1	671121	675824	CHM13_T0000196	0	-	671216	675792	0,0,0	8	194,79,91,109,152,85,117,209,	0,382,609,991,1501,1738,1928,4494,
chr1	678944	680502	CHM13_T0000199	0	-	680502	680502	0,0,0	1	1558,	0,
