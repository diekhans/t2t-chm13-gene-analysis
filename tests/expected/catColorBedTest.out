coding 0,0,128 navy
lncRNA 0,100,0 darkgreen
pseudo 255,0,255 fuchsia
smallRNA 218,165,32 goldenrod
unsure 138,43,226 blueviolet
